packages/react-devtools-shared/src/devtools/views/Profiler/ChartNode.js

```diff
@@ -41,12 +45,13 @@ export default function ChartNode({
 }: Props) {
   return (
     <g className={styles.Group} transform={`translate(${x},${y})`}>
-      <title>{label}</title>
       <rect
         width={width}
         height={height}
         fill={color}
         onClick={onClick}
+        onMouseOver={onMouseOver}
+        onMouseOut={onMouseOut}
```

Comment:
nit: Not sure it matters in this case (SVG rect) but in general, I'd prefer using mouse-enter and mouse-leave rather than mouse-over and mouse-out

Reply:
Yeah, agree, I changed them to `onMouseLeave/Enter`, I think we also don't need to `stopPropagation` anymore, I removed them too:)